### DISCUSSION TITLE
Other Command Values

### DISCUSSION THREAD

**Discussion by TheRingland** [KrystianD/bm2-battery-monitor#13] [Q&A]
I don’t suppose you decompiled the other functions like the crank test data or historical data log extraction, or accidentally left the jar file hanging around for someone else to do it?

**Comment by KrystianD** [comment]
Hi @TheRingland, I decompiled the whole code, including other commands as well. However, I didn't need historical data - current values were enough for me. But with enough effort it would be possible to implement fetching crank test data and history. It is not easy to follow the code, given the way it was written, but doable.

If you want to give it a go, ping me over any [DM channel](https://github.com/KrystianD#speech_balloon-want-to-talk-about-my-projects-you-have-something-to-be-done) and I can share decompilation results.
